src/Connection/Cluster/RedisCluster.php

```diff
@@ -54,6 +56,7 @@ class RedisCluster implements ClusterInterface, \IteratorAggregate, \Countable
     private $strategy;
     private $connections;
     private $retryLimit = 5;
+    private $minRetryAfter = 1; // seconds
```

Comment:
Let's call it `$retryAfter` and make it milliseconds, seconds is too coarse.

Reply:
its a float : you can use 0.2 if you want (usleep is used)

Reply:
I used the name $minRetryAfter to emphasize that it is a starting value and not a fixed value: the value is incremented exponentially.